papaparse.js

```diff
@@ -1457,6 +1457,34 @@ License: MIT
 			if (!input)
 				return returnable();
 
+			// Rename headers if there are duplicates
+			if (config.header)
+			{
+				let firstLine = input.split(newline)[0]
+				let separator = '_';
+				let headers = firstLine.split(delim) 
+				let header_map = []; 
+				let header_count = {};
+
+				for (var i in headers){
```

Comment:
I guess it will be easier to read if you write: 

var header = headers[i]; 

and then use `header` instead of `headers[i]` in all of the loop code.